frontend/sw360-portlet/src/main/webapp/html/vulnerabilities/view.jsp

```diff
@@ -35,7 +35,7 @@
         <thead>
         <tr>
             <th class="infoheading">
-                Display Filter
+                Quick Filter
```

Comment:
Yes, for consistency. Also, #506 will consolidate the snippet to one jspf anyway.